### DISCUSSION TITLE
Open Alpha only when nvim opens a whole directory

### DISCUSSION THREAD

**Discussion by MrRoiz** [goolord/alpha-nvim#203] [Q&A] [answered]
Hey there, I'm kinda new on nvim, I recently installed alpha-nvim and it works fine this is my config 
![image](https://user-images.githubusercontent.com/49358353/232552032-64d14ed7-edeb-4053-bc60-0f53dcd4cc61.png)

Alpha opens up whenever nvim is open, but is it possible to avoid open it when it opens a single file?
eg. 
```bash
nvim init.lua
```

**Comment by goolord** [comment]
just remove `alpha.start()`. there's already a bunch of conditions set for alpha to only open when it's sensible

**Reply by MrRoiz** [reply]
without `alpha.start()` it does not open on startup 😥

**Reply by goolord** [reply, marked as answer]
https://github.com/goolord/alpha-nvim/blob/dafa11a6218c2296df044e00f88d9187222ba6b0/lua/alpha.lua#L539-L568

here's the conditions for alpha automatically starting. in my experience providing support for like a dozen other people asking the exact same question as you it's either been

1. some shell configuration aliasing `nvim` with extra arguments, or someone simply starting up nvim like `nvim .` or something
2. a configuration problem with their package manager (usually lazy loading the plugin or something)

**Reply by MrRoiz** [reply]
You were totally right, my bad, I had an alias for nvim I just forgot. I'm pretty new on this, trying to improve, thanks! and awesome plugin btw ❤️
